distributed/client.py

```diff
@@ -359,6 +358,13 @@ def traceback(self, timeout=None, **kwargs):
     def type(self):
         return self._state.type
 
+    @property
+    def _typ(self):
```

Comment:
Nitpick: Should we call this `_type` instead? Adding the `e` at the end might make this slightly more clear for readers

Reply:
Done thanks!